insights/tests/__init__.py

```diff
@@ -178,7 +178,7 @@ def integrate(input_data, module):
     This method drives the mapreduce framework with test `input_data`.
 
     :param InputData input_data: InputData object filled with test data
-    :param module module: module containing parsers and reducers to test with
```

Comment:
Not sure this is relevant to this merge request :-)

Reply:
@PaulWay - There are two `module`s declared in docstring (parser.Krb5Configuration.module) and (combiner.AllKrb5Conf.module), but here none is referenced. And it will break making the document. I think, here, the `module` in line-180 is not a `type` used in the core, and it would point to the `plugin`, so I modified it. And how do you think this modification @bfahr.

Reply:
As per the discussion with @bfahr, I separate the modification of `test/__init__.py` and `parsers/krb5.py` to another 2 PRs, since they are not the part of the update of `combiners/krb5.py`

See #452 and #453.